### PR TITLE
add polygon query

### PR DESCRIPTION
**Pull request by wavemechanics**
### What

Add polygon= query and cli parameters.

### CODE REVIEW

File: pkg/geodata/geodata.go [hunk 1]
```diff
@@ -227,6 +230,77 @@ AND nomis_category.year = %d
...
+    ),
+    geo.wkb_geometry
+)
+AND geo.valid
+AND geo_type.id = geo.type_id
+%s
+AND geo_metric.geo_id = geo.id
+AND data_ver.id = geo_metric.data_ver_id
+AND data_ver.census_year = %d
+ANd data_ver.ver_string = '2.2'
```
Comment (VivianAllen) [inline pkg/geodata/geodata.go]: OOS for this, but should this data_ver value be pulled from config / the query?

File: pkg/geodata/polygon.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+func (c *Point) String() string {
+	return fmt.Sprintf("%.13g %.13g", c.Lon, c.Lat)
+}
+
+func ParsePolygon(s string) ([]Point, error) {
+	a := strings.Split(s, ",")
+	if len(a)%2 != 0 {
+		return nil, errors.New("polygon must have even number of coordinates")
```
Comment (VivianAllen) [inline pkg/geodata/polygon.go:22]: is this true? what about a closed square?

Comment (VivianAllen) [reply pkg/geodata/polygon.go:22]: ahh hang on I've misunderstood - coordinates come in pairs

Comment (VivianAllen) [reply pkg/geodata/polygon.go:22]: my bad. Ignore

**Review by VivianAllen — DISMISSED** [review]
looks good, tests are nice. As discussed offline possibly need some tests involving an actual test-fixture database for this, but OOS for this PR. ✅

File: pkg/geodata/geodata.go (hunk 1 above)
Comment (wavemechanics) [reply pkg/geodata/geodata.go]: Yes. I guess we can add options when we eventually add more versions.

**Review by VivianAllen — APPROVED** [review]

**Pull request merged**